### ISSUE TITLE
Instagram Multiple scopes

### ISSUE THREAD

**Issue by mrgodhani** [thephpleague/oauth2-client#335]
Hi

How would I set multiple scopes for Instagram provider ?

This is what I have :

```
'instagram' => [
        'clientId'      => env('INSTAGRAM_CLIENTID'),
        'clientSecret'  => env('INSTAGRAM_CLIENTSECRET'),
        'redirectUri'   => env('INSTAGRAM_REDIRECTURI'),
        'scopes'        => ['likes','basic']
    ]
```

And this is the error I get :
```
{"code": 400, "error_type": "OAuthException", "error_message": "Invalid scope field(s): likes,basic"}
```

**Comment by rtheunissen** [comment]
I think the Instagram scope separator should be `+`. See [docs](https://instagram.com/developer/authentication/#scope)

**Comment by rtheunissen** [comment]
PR to fix this has been opened [here](https://github.com/thephpleague/oauth2-client/pull/336). @ramsey

**Comment by rtheunissen** [comment]
@mrgodhani I have opened an issue on [thephpleague/oauth2-instagram](https://github.com/thephpleague/oauth2-instagram/issues)

**Comment by ramsey** [comment]
The scope separator for Instagram has been fixed in 0.12.0. Please let me know if that fixes this issue.

**Comment by mrgodhani** [comment]
:+1:  I will check it out today and let you know

**Comment by mrgodhani** [comment]
```
{"code": 400, "error_type": "OAuthException", "error_message": "Invalid scope field(s): basic+likes+comments+relationships"}
```
Still get this error

**Comment by stevenmaguire** [comment]
I will take care of this and submit a PR.

**Comment by stevenmaguire** [comment]
I've submitted a PR to correct the scope separator. Luckily I found that the solution also applies to the LinkedIn provider. I've submitted a separate PR for that. 

Additionally, both of these providers have been updated in their broken out packages.

**Comment by ramsey** [comment]
The fix for this has been released in 0.12.1.
